tfjs-backend-webgpu/src/kernels/binary_op_util.ts

```diff
@@ -143,7 +143,7 @@ const POW_VEC4 = `
   if (isExpZero.a) {
     resultTemp.a = 1.0;
   }
-  let isNaN = vec4<f32>(a < vec4<f32>(0.0)) * vec4<f32>(floor(b) < b);
+  let isNaN = a < vec4<f32>(0.0) & floor(b) < b;
```

Comment:
Why the check of `isNaN` is different with L161?

Reply:
`a` and `b` are the different parameters for `pow` and `minumin/maxmium`, I think these two operators have different behaviors for NaN.